UITests/Responses/TransferFunds/AvailableFundZero.json

```diff
@@ -0,0 +1,24 @@
+{
+    "token": "trf-token",
+    "status": "QUOTED",
+    "createdOn": "2019-07-24T18:49:41",
+    "clientTransferId": "6712348070814",
+    "sourceToken": "usr-token",
```

Comment:
You will get the same error as the above one if there is zero or insufficient funds.  The only exception is when there is 0 funds and you attempt to transfer all funds.  It will return this response
```
{
  "errors" : [ {
    "message" : "Amount is less than the fee amount",
    "code" : "AMOUNT_LESS_THAN_FEE"
  } ]
}
```